src/app/routes/query.py

```diff
@@ -1,6 +1,12 @@
+import os
+import re
 from uuid import uuid4
 from flask import render_template, request, flash, redirect, url_for
 from click.testing import CliRunner
+from cryptography.fernet import Fernet
+from hdfs import InsecureClient
+from werkzeug.utils import secure_filename
+
```

Comment:
What's with these new imports?

Reply:
probably a merge artifact. I will remove the unused ones.

Reply:
fixed